test/verify/check-kdump

```diff
@@ -178,15 +178,15 @@ class TestKdump(KdumpHelpers):
         b.wait_text("#kdump-target-info", "Remote over SSH, root@localhost:/var/crash")
 
         # try to change the path to a directory that doesn't exist
-        customPath = "/var/crash2"
+        customPath = "/crash2"
```

Comment:
Hmm, does that mean that the default location `/var/crash` doesn't work either? Or is that its own  subvolume/mount?

Reply:
> Hmm, does that mean that the default location `/var/crash` doesn't work either?

Correct.

Reply:
Thanks for confirming -- (not) cool. We didn't invent that location, it's the kdump default. So that'll be a Fedora bugzilla + naughty?

Reply:
I filed the bug: https://bugzilla.redhat.com/show_bug.cgi?id=2306035

But why not work around in our test?

Reply:
Oh, it's fine to work around, I was just asking for it to be reported. Thanks!